cmd/analyzer/parse_args.go

```diff
@@ -56,11 +60,13 @@ const (
 	VPC                   = "vpc"
 	Debug                 = "debug"
 	Version               = "version"
-	QProtocol             = "q-protocol"
-	QSrcMinPort           = "q-src-min-port"
-	QSrcMaxPort           = "q-src-max-port"
-	QDstMinPort           = "q-dst-min-port"
-	QDstMaxPort           = "q-dst-max-port"
+	ESrc                  = "e-src"
+	EDst                  = "e-dst"
+	EProtocol             = "e-protocol"
+	ESrcMinPort           = "e-src-min-port"
+	ESrcMaxPort           = "e-src-max-port"
+	EDstMinPort           = "e-dst-min-port"
+	EDstMaxPort           = "e-dst-max-port"
```

Comment:
suggest renaming cli args, remove prefix "e-", for example "src" instead of "e-src"